src/wp-includes/class-wp-query.php

```diff
@@ -1410,6 +1414,32 @@ protected function parse_search( &$q ) {
 		$searchand                 = '';
 		$q['search_orderby_title'] = array();
 
+		$default_search_columns = array( 'post_title', 'post_excerpt', 'post_content' );
+		$search_columns         = ! empty( $q['search_columns'] ) ? $q['search_columns'] : $default_search_columns;
+		if ( ! is_array( $search_columns ) ) {
+			$search_columns = array( $search_columns );
+		}
+
+		/**
+		* Filters the columns to search in a WP_Query search.
+		*
+		* The supported columns are `post_title`, `post_excerpt` and `post_content`.
+		* They are all included by default.
+		*
+		* @since 6.2.0
+		*
+		* @param string[] $search_columns Array of column names to be searched.
+		* @param string   $search         Text being searched.
+		* @param WP_Query $this           The current WP_Query instance.
+		*/
```

Comment:
```suggestion
		/**
		 * Filters the columns to search in a WP_Query search.
		 *
		 * The supported columns are `post_title`, `post_excerpt` and `post_content`.
		 * They are all included by default.
		 *
		 * @since 6.2.0
		 *
		 * @param string[] $search_columns Array of column names to be searched.
		 * @param string   $search         Text being searched.
		 * @param WP_Query $this           The current WP_Query instance.
		 */
```
Looks like a slight alignment issue here.